src/Pages/DonorPage/index.js

```diff
@@ -1,126 +1,87 @@
 import React, { useEffect, useState } from 'react';
-import { v4 } from 'uuid';
-
 import BreadCrumb from '../../Components/BreadCrumb';
 import Navbar from '../../Components/Navbar';
 import Search from '../../Components/Search';
-import Paginator from '../../Components/Paginator';
-import Status from '../../Components/Status';
-
 import styles from './style.module.css';
-
-// TODO: Pull real data, not dummy data
-const testData = [
-  {
-    name: 'Zach Gallaway',
-    businessName: 'Food 4 U',
-    dateSubmitted: '2023/01/19',
-    status: 'pending',
-  },
-  {
-    name: 'Jason Derulo 2',
-    businessName: 'Autotunes, Inc',
-    dateSubmitted: '2023/01/17',
-    status: 'pending',
-  },
-  {
-    name: 'Jason Derulo 3',
-    businessName: 'Autotunes, Inc',
-    dateSubmitted: '2023/01/17',
-    status: 'pending',
-  },
-  {
-    name: 'Jason Derulo 4',
-    businessName: 'Autotunes, Inc',
-    dateSubmitted: '2023/01/17',
-    status: 'pending',
-  },
-  {
-    name: 'Jason Derulo 5',
-    businessName: 'Autotunes, Inc',
-    dateSubmitted: '2023/01/17',
-    status: 'pending',
-  },
-  {
-    name: 'Jason Derulo 6',
-    businessName: 'Autotunes, Inc',
-    dateSubmitted: '2023/01/17',
-    status: 'pending',
-  },
-  {
-    name: 'Jason Derulo 7',
-    businessName: 'Autotunes, Inc',
-    dateSubmitted: '2023/01/17',
-    status: 'pending',
-  },
-  {
-    name: 'Jason Derulo 8',
-    businessName: 'Autotunes, Inc',
-    dateSubmitted: '2023/01/17',
-    status: 'pending',
-  },
-  {
-    name: 'Jason Derulo 9',
-    businessName: 'Autotunes, Inc',
-    dateSubmitted: '2023/01/17',
-    status: 'pending',
-  },
-  {
-    name: 'Jason Derulo 10',
-    businessName: 'Autotunes, Inc',
-    dateSubmitted: '2023/01/17',
-    status: 'pending',
-  },
-  {
-    name: 'Jason Derulo 11',
-    businessName: 'Autotunes, Inc',
-    dateSubmitted: '2023/01/17',
-    status: 'pending',
-  },
-  {
-    name: 'Jason Derulo 12',
-    businessName: 'Autotunes, Inc',
-    dateSubmitted: '2023/01/17',
-    status: 'pending',
-  },
-];
+import { GetDonors } from '../../Services/DonorsService';
+import { DataTable, Pagination } from '../../Components/DataTable';
 
 function DonorPage() {
-  const entriesPerPage = 10;
-  const [displayData, setDisplayData] = useState([]);
+  const defaultPageSize = 8;
+  const [donors, setDonors] = useState([]);
+  const [sortColumn, setSortColumn] = useState({ path: 'no', order: 'asc' });
```

Comment:
@sherviniv could you please change this to reflect what the variable names will be 
should be  `order` and `sort_by`

see open sort PR here:
https://github.com/FoodIsLifeBGP/banana-rails/pull/168/files#r1375314653